test/Microsoft.AspNetCore.Mvc.IntegrationTests/BodyValidationIntegrationTests.cs

```diff
@@ -297,10 +294,11 @@ public class BodyValidationIntegrationTests
             Assert.False(modelState.IsValid);
             var modelStateErrors = CreateValidationDictionary(modelState);
             Assert.Equal(2, modelStateErrors.Count);
-            Assert.Equal("The field Price must be between 100 and 200.", modelStateErrors["Price"]);
-            Assert.Equal(
-                "The field Contact must be a string with a maximum length of 10.",
-                modelStateErrors["Contact"]);
+
+            var priceRange = ValidationAttributeUtil.GetRangeErrorMessage(100, 200, "Price");
+            var contactLength = ValidationAttributeUtil.GetStringLengthErrorMessage(null, 10, "Contact");
```

Comment:
Not a big deal but these two lines are part of `// Arrange`. Similar thoughts elsewhere with same low priority.